### DISCUSSION TITLE
How can edit the indicator in craft js?

### DISCUSSION THREAD

**Discussion by xs-nayeem** [prevwong/craft.js#584] [Q&A]
When I drag an element, the indicator shows a color where it can be dropped or not. But I want to make a space between elements when I am dragging an element between two elements. 

Please check the image for reference.

![Screenshot_5](https://github.com/prevwong/craft.js/assets/106504976/ce8211ef-f3ed-4088-a6d8-78285458891b)

Hoping for a solution.

**Comment by MrVibe** [comment]
use renderNode funciton in your craftJs instance

**Reply by loctp2003** [reply]
Can you give example for this

**Reply by MrVibe** [reply]
You have to utilise this : node.events.hovered , rendernode is the best place as it is a wrapper on all the nodes which is used to display the outline and title on the elements.

**Reply by loctp2003** [reply]
how to use rendernode
